views/marts/product/fct_active_users.view.lkml

```diff
@@ -76,31 +76,86 @@ view: fct_active_users {
   dimension: daily_desktop_active_users {
     type: number
     sql: ${TABLE}.daily_desktop_active_users ;;
-    label: "Desktop/Client DAU"
-    description: "DAU reported by Rudderstack telemetry."
+    label: "Desktop DAU"
+    description: "Desktop DAU reported by telemetry."
   }
 
   measure: total_daily_desktop_active_users {
     type: sum
     sql: ${daily_desktop_active_users} ;;
-    label: "Total Desktop/Client DAU"
+    label: "Total Desktop DAU"
     view_label: "  * Metrics. DAU"
   }
 
-  dimension: daily_legacy_active_users {
+  dimension: daily_webapp_active_users {
     type: number
-    sql: ${TABLE}.daily_legacy_active_users ;;
-    label: "Legacy Desktop/Client DAU"
-    description: "DAU reported by Segment telemetry."
+    sql: ${TABLE}.daily_webapp_active_users ;;
+    label: "Webapp DAU"
+    description: "Webapp DAU reported by telemetry."
+  }
+
+  measure: total_daily_webapp_active_users {
+    type: sum
+    sql: ${daily_webapp_active_users} ;;
+    label: "Total Webapp DAU"
+    view_label: "  * Metrics. DAU"
   }
 
-  measure: total_daily_legacy_active_users {
+  dimension: daily_client_desktop_active_users {
+    type: number
+    sql: ${TABLE}.daily_client_desktop_active_users ;;
+    label: "Client Desktop DAU"
+    description: "Client DAU reported by Rudderstack telemetry."
+  }
+
+  measure: total_daily_client_desktop_active_users {
     type: sum
-    sql: ${daily_legacy_active_users} ;;
-    label: "Total Legacy Desktop/Client DAU"
+    sql: ${daily_client_desktop_active_users} ;;
+    label: "Total Client Desktop DAU"
     view_label: "  * Metrics. DAU"
   }
 
+  dimension: daily_legacy_desktop_active_users {
+    type: number
+    sql: ${TABLE}.daily_legacy_desktop_active_users ;;
+    label: "Legacy Desktop DAU"
+    description: "Desktop DAU reported by Segment telemetry."
+  }
+
+  measure: total_daily_legacy_desktop_active_users {
+    type: sum
+    sql: ${daily_legacy_desktop_active_users} ;;
+    label: "Total Legacy Desktop DAU"
+    view_label: "  * Metrics. DAU"
+  }
+
+  dimension: daily_client_webapp_active_users {
```

Comment:
Is this different from `daily_webapp_active_users`? If yes, label should be explicit. If not, only one should be kept.

Reply:
Added all the possible combinations to the above comment.